test/unit/test-amp-story-player.js

```diff
@@ -691,6 +691,88 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
       return expect(() => player.go(-1)).to.throw('Out of Story range.');
     });
 
+    it('last story call to next_() is the first story', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+      await nextTick();
+
+      const navigationSpy = env.sandbox.spy();
+      playerEl.addEventListener('navigation', navigationSpy);
+      playerEl.setAttribute('enable-circular-wrapping', true);
```

Comment:
Here and below, so we stick to our own API spec

```suggestion
      playerEl.setAttribute('enable-circular-wrapping', '');
```